setup.py

```diff
@@ -30,7 +27,12 @@
     license="MIT",
     packages=setuptools.find_packages("src"),
     package_dir={"": "src"},
-    install_requires=requirements,
+    install_requires=[
+        "requests",
+        "urllib3",
+        "waiting",
+        "python-dateutil",
+    ],
     tests_require=requirements + test_requirements,
```

Comment:
requirements is also defined here

Reply:
oops you're right
returning the definition of requirements from ``requirements.txt`` and using it for the ``test_require`` (it seams fit to pin it there)